lib/crystalball/source_diff.rb

```diff
@@ -28,7 +28,18 @@ def empty?
 
     # TODO: Include untracked to changeset
     def changeset
-      @changeset ||= git_diff.map { |file_diff| FileDiff.new(file_diff) }
+      @changeset ||= git_diff.map do |file_diff|
```

Comment:
You could just use `flat_map`

Reply:
Right! But some specs failed :thinking: I need to explore it a bit

Reply:
no need for this flat_map anymore